node/include/cocaine/detail/service/node/dispatch/handshake.hpp

```diff
@@ -9,11 +9,12 @@
 
 #include "cocaine/idl/rpc.hpp"
 
-#include "cocaine/service/node/slot.hpp"
+#include "cocaine/detail/service/node/forwards.hpp"
```

Comment:
That makes sense, however it is not consistent with cocaine style. We need to make an agreement, that in all new code we will use that new convention.